recipes/cssmin/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "cssmin" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  sha256: 0649244b431c8d604d443047c66156cf5b0d0677c6beacbdaf40594ee2fd4ea4
+  url: https://github.com/zacharyvoase/cssmin/archive/v{{ version }}.tar.gz
+
+build:
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+  number: 0
+  entry_points:
+    - cssmin = cssmin:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  commands:
+    - cssmin --help
+
+about:
+  license_family: BSD
+  license: BSD
```

Comment:
Can you determine which flavor of BSD license this is and then use `BSD-3-Clause`, `BSD-2-Clause` etc, whichever is appropriate.

Reply:
I think that the Python package cssmin is actually licensed under the MIT-license, but contains code taken from the "YUI CSS Compressor" software, which is licensed under the BSD 3-Clause license.

See: https://github.com/zacharyvoase/cssmin/blob/master/LICENSE

Reply:
You think its MIT then? The source code does not explicitly state which parts are from the "YUI CSS Compressor"..

Reply:
I don't consider myself a licensing expert, but as far as I understand it, the BSD-3-Clause allows you to use and relicense the code as long as you keep the original copyright notice with it. So in this case the author took BSD-3-Clause licensed code, added their own code, and relicensed everything under the MIT license. They complied with the BSD license by adding the original copyright notice to the new copyright notice.

This package should be classified as licensed under the MIT license.

See also: https://www.patrick-wied.at/blog/how-to-correctly-use-code-you-didnt-write